pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -52,6 +52,7 @@ let
     , extraPoliciesFiles ? []
     , libName ? browser.libName or "firefox" # Important for tor package or the like
     , nixExtensions ? null
+    , hasMozSystemDirPatch ? (lib.hasPrefix "firefox" pname && lib.hasSuffix "-bin" pname)
```

Comment:
Should not this be `!lib.hasSuffix "-bin" pname`?

Reply:
derp 🤦🏼